src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    evacuateCoords: {
+      x: 1400,
+      y: 500,
+    },
+
+    evacuate() {
+      this.coords.x = this.evacuateCoords.x;
+      this.coords.y = this.evacuateCoords.y;
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(value = 1) {
```

Comment:
```suggestion
    get info() {
      return `name: ${this.name}, chip version: ${this.version}, `
      + `wheels: ${this.wheels}`;
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    goForward(value = 1) {
```
add spaces between blocks of code